hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/create/ClassicCreatesCall.java

```diff
@@ -180,9 +180,6 @@ private ResponseCodeEnum validityOfSynth(@NonNull final TokenCreateTransactionBo
         if (treasuryAccount == null) {
             return INVALID_ACCOUNT_ID;
         }
-        if (op.autoRenewAccount() == null) {
```

Comment:
This check was removed because in Hedera Docs auto renew account is optional and before removing this checks the tests were failing. 
Can you please share your opinion if this check is needed, @tinker-michaelj @Neeharika-Sompalli ?

Reply:
@fchitakova Could you please point me to the test that was failing ?

Reply:
The preivous failing tests were `createTokenWithDefaultExpiryAndEmptyKeys` and `createTokenWithFixedFeeThenTransferAndAssessFee`

Reply:
Also, in documentation is pointed that auto renew account is optional and it should not fail if it is not provided

Reply:
yes this also make sense to me.